src/api/currency/content-types/currency/schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "kind": "collectionType",
+  "collectionName": "currencies",
+  "info": {
+    "singularName": "currency",
+    "pluralName": "currencies",
+    "displayName": "Financial.CurrencyConversion"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "year": {
+      "type": "string",
+      "default": "2020",
+      "required": true,
+      "maxLength": 4
+    },
+    "month": {
+      "type": "integer",
+      "default": 1,
```

Comment:
**question:** Same as above, do we need to have a default value here?

Reply:
Questioned that myself, put a default to avoid any "01". Will change this to hold no default if it's the preference.

Reply:
Removed the default value.